zanata-war/src/main/java/org/zanata/action/CopyTransAction.java

```diff
@@ -77,6 +80,7 @@
     private org.zanata.ui.faces.FacesMessages jsfMessages;
 
     @Inject
+    @Any
```

Comment:
Probably a left over. I'll try removing it.

On 1 March 2016 at 09:42, Carlos Munoz <notifications@github.com> wrote:

> In zanata-war/src/main/java/org/zanata/action/CopyTransAction.java
> <https://github.com/zanata/zanata-server/pull/1072#discussion_r54499751>:
>
> > @@ -77,6 +80,7 @@
> >      private org.zanata.ui.faces.FacesMessages jsfMessages;
> >
> >      @Inject
> > +    @Any
>
> why is there a need for @Any here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1072/files#r54499751>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat